pkg/vmimage/data/script.sh

```diff
@@ -36,7 +36,7 @@ cat >client-key.pem <<'EOF'
 {{ .Builder.ClientKey | PrivateKeyAsBytes | String }}
 EOF
 
-go get github.com/jim-minter/tlsproxy
+go get github.com/openshift/openshift-azure/cmd/
```

Comment:
vmimage job seems to failing on this step, and this also fails when I run it locally. Should this be `go get github.com/openshift/openshift-azure/cmd/azure`? (seems to be the only cmd entrypoint that references tlsproxy)

Reply:
I think so too. tlsproxy is part of `azure` binary subcommands